packages/core/lib/match.js

```diff
@@ -42,16 +45,56 @@ function match (test, options, configSetters) {
 
 const regexify = glob => globToRegex(glob, { extended: true })
 
-function createFileTypeMatcher (test) {
-  if (typeof test === 'string') {
+// split positive matches and negative matches
+function splitPatterns (patterns) {
```

Comment:
Why doesn't it just return `[ inclusions, exclusions ]`? 😉

Reply:
Or `{ inclusions, exclusions }`.

Reply:
`{ inclusions, exclusions }` looks better!